business-registry-dashboard/app/enums/authorized-actions.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * List of actions authorized via roles. See Authorizations module.
+ * Ultimately, this will be combined into the Authorization Roles enum.
+ */
+export enum AuthorizedActions {
+  ADD_ENTITY_NO_AUTHENTICATION,
+  STAFF_CREATE_AFFILIATION,
+  STAFF_CREATE_NR_AFFILIATION,
+  STAFF_DASHBOARD,
+  STAFF_LOAD_AFFILIATIONS,
+  STAFF_REMOVE_BUSINESS,
+  STAFF_RESTORE_OR_REINSTATE
```

Comment:
Other than STAFF_DASHBOARD (which literally displays "My Staff Business Registry"), can the others be de-coupled from staff?

Reply:
Looks great!

Can you please provide a short description of the authorized actions? You can add them to the spreadsheet directly if you want: https://docs.google.com/spreadsheets/d/13P4ZA19lJfXJX95N5OfQMu6jYuotDH6h2hDf2DnAv0E/edit?gid=1335805235#gid=1335805235.